src/types/modules/window.d.ts

```diff
@@ -0,0 +1,10 @@
+declare global {
```

Comment:
You can use `src/types/global.d.ts` for this 😄

Reply:
@blazejkustra Updated!